service/build.gradle

```diff
@@ -158,6 +166,9 @@ dependencies {
   compile group: 'uk.gov.hmcts.reform', name: 'pdf-generator', version: '1.0.2'
 
   compile group: 'commons-io', name: 'commons-io', version: '2.6'
+  compile group: 'org.pitest', name: 'pitest', version: '1.3.2'
+  compile group:'info.solidsoft.gradle.pitest', name: 'gradle-pitest-plugin', version: '1.3.0'
```

Comment:
Plugins should be declared in plugins section. Move it.

Reply:
These are dependencies.

Reply:
Looks like a plugin.